pkg/apis/fleet/v1alpha1/types.go

```diff
@@ -248,6 +250,72 @@ type PodSecurityPolicy struct {
 	ValidationFailureAction string `json:"validationFailureAction,omitempty"`
 }
 
+// BackupConfig defines the configuration for the Velero backup engine.
+type BackupConfig struct {
+	// Chart defines the helm chart config of the velero.
+	// default values is
+	//
+	// chart:
+	//   repository: https://vmware-tanzu.github.io/helm-charts
+	//   name: velero
+	//   version: 5.0.2
+	//
+	// +optional
+	Chart *ChartConfig `json:"chart,omitempty"`
+
+	// Storage details where the backup data should be stored.
+	Storage BackupStorage `json:"storage"`
+
+	// VeleroImage Details of the container image to use in the Velero deployment & daemonset
+	// default values is
+	//
+	//  repository: velero/velero
+	//  tag: v1.11.1
+	//  pullPolicy: IfNotPresent
+	//
+	// +optional
+	VeleroImage apiextensionsv1.JSON `json:"veleroImage,omitempty"`
+
+	// InitContainers to add to the Velero deployment's pod spec.
+	// default values is
+	//
+	// - name: velero-plugin-for-aws
+	//   image: velero/velero-plugin-for-aws:v1.7.1
+	//   imagePullPolicy: IfNotPresent
+	//   volumeMounts:
+	//     - mountPath: /target
+	//       name: plugins
+	//
+	// +optional
+	InitContainers apiextensionsv1.JSON `json:"initContainers,omitempty"`
```

Comment:
Not sure i understand what does this container do

Reply:
this container is a required setting for installing Velero. Perhaps we can remove this user-specific setting and use the default instead.